README.md

```diff
@@ -9,20 +9,22 @@ It provides a way to do:
 
 You will need to set the `requireSession` and `editOnly` values to `true` in etherpad's `settings.json` file.
 
-It's recommended to also add in a `sessionKey`. This can be any random value, but should be the same across the cluster.
```

Comment:
When I had it in the settings.json I got this deprecation warning:

    [2015-06-10 09:21:46.175] [WARN] console - Declaring the sessionKey in the settings.json is deprecated. This value is auto-generated now. Please remove the setting from the file.